CHANGELOG.md

```diff
@@ -1,3 +1,61 @@
+# Musicoin Desktop Wallet 1.0
+
+### Sleek new UI:
+
+1. An all new UI with a new quick access navbar, from where critical functionalities like signing messages, checking exchange balance, direct links to Musicoin's public forums along with an introduction section only on first start-up for guiding users new to how a wallet functions.
+
+2. Unique profile picture assigned to each account in order to make identification easier.
+
+3. Standalone Send Funds button so that you can send funds from any of your accoutns, with one click
+
+4. Integrated Music Player - Access musicoin.org from within the wallet UI while it syncs and transfer coins easily, efficiently and without errors between the web and desktop wallets. You can also maximize/minimize the windows to enhance your music listening experience.
+
+5. Clear indication when the wallet is in sync - No more worrying whether your tx made it through or not, the send funds button unlocks only when the wallet is within a few blocks of the chain
+
+6. View your balance in btc, usd and MUSIC at the click of a button, thanks to a new clutter free design, which introduces jsut three columns showing you critical info.
+
+7. Per account backups and import function
+
+### Reduced weight:
+
+Improved fast sync in complaince with go-ethereum has reduced the chain size to < 10 GB
+
+### Improved synchronisation and security:
+
+1. Previosuly, the list of bootstrap nodes was not loaded on start-up, now it does. Users on connecting to a new peer will save new bootstrap nodes in `bootnodes.json` automatically.
+
+2. User also has ability to restore default smaller nodes list via menu (Advanced - Restore default nodes list)
+
+3. In case your local time is incorrect, you would see notification, with suggestion to enable time synchronisation in your OS.
+
+4. Displaying private keys and mnemonics don't store state providing good privacy
+
+### Host of configurable options:
+
+1. Select go-musicoin cache size from the new navbar at the top of the screen.
+
+2. Restore the default list of enodes, if you prefer connecting to a set of trusted peers.
+
+3. Sign and verify messages, all from within the wallet itself.
+
+4. Create a wallet wiht an easy to remember mnemonic or choose the standard way of using a password.
+
+5. Display the private key at the click of a button, forget your kes no more.
```

Comment:
Misstype. Also might be worth to add paper wallet after all.

Reply:
Ah, nice catch. Lets add the paper wallet to release-1.5 ?(see my message over at slack). I'll merge this once the press release is out for the community to see.

Reply:
Yes, I see you merged to 1.5. I switched builder to 1.5. So paper wallet should be in the build.